debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -509,25 +510,45 @@ private static int validateHosts(Configuration config, Field field, ValidationOu
     }
 
     private static int validateCollectionExcludeList(Configuration config, Field field, ValidationOutput problems) {
-        String includeList = config.getFallbackStringProperty(COLLECTION_INCLUDE_LIST, COLLECTION_WHITELIST);
-        String excludeList = config.getFallbackStringProperty(COLLECTION_EXCLUDE_LIST, COLLECTION_BLACKLIST);
+        String includeList = config.getString(COLLECTION_INCLUDE_LIST);
+        String excludeList = config.getString(COLLECTION_EXCLUDE_LIST);
         if (includeList != null && excludeList != null) {
             problems.accept(COLLECTION_EXCLUDE_LIST, excludeList, COLLECTION_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG);
             return 1;
         }
         return 0;
     }
 
+    private static int validateCollectionBlacklist(Configuration config, Field field, ValidationOutput problems) {
+        String whitelist = config.getFallbackStringPropertyWithWarning(COLLECTION_INCLUDE_LIST, COLLECTION_WHITELIST);
+        String blacklist = config.getFallbackStringPropertyWithWarning(COLLECTION_EXCLUDE_LIST, COLLECTION_BLACKLIST);
+        if (whitelist != null && blacklist != null) {
+            problems.accept(COLLECTION_EXCLUDE_LIST, blacklist, COLLECTION_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG);
+            return 1;
+        }
+        return 0;
+    }
+
     private static int validateDatabaseExcludeList(Configuration config, Field field, ValidationOutput problems) {
-        String includeList = config.getFallbackStringProperty(DATABASE_INCLUDE_LIST, DATABASE_WHITELIST);
-        String excludeList = config.getFallbackStringProperty(DATABASE_EXCLUDE_LIST, DATABASE_BLACKLIST);
+        String includeList = config.getString(DATABASE_INCLUDE_LIST);
+        String excludeList = config.getString(DATABASE_EXCLUDE_LIST);
```

Comment:
I might be wrong on this, not 100% confident, but:
Isn't that check already handled by `#validateDatabaseBlacklist()`?
```
String whitelist = config.getFallbackStringPropertyWithWarning(DATABASE_INCLUDE_LIST, DATABASE_WHITELIST);
String blacklist = config.getFallbackStringPropertyWithWarning(DATABASE_EXCLUDE_LIST, DATABASE_BLACKLIST);
```
Variables `whitelist` and `blacklist` will both have the new field's values and return the error for the `*_BLACKLIST` field (which is wrong by the way, when it did not come from the fallback but the new field, right?)

Like I suggested in the previous PR and discusscions I think it's best to only assign ONE validator to one of those fields of a group and set the error respectively on that field?
Example: There are 4 fields for each group (`DATABASE_WHITELIST`, `DATABASE_INCLUDE_LIST`, `DATABASE_BLACKLIST`, `DATABASE_EXCLUDE_LIST`). To avoid duplicate output of warnings (which is imo solved for the fallback case) AND putting duplicate warnings for the same problem on different fields (where only putting it on one specific field is valid) I suggest:

Remove all these validators.
Add a new validator to only one of those fields in the group, e.g. "*_INCLUDE_LIST`.
That validator checks:
- if legacy AND new field for includes is set (yields the warning log msg)
- if legacy AND new field for excludes is set (yields the warning log msg)
- if any include AND any exclude is set (adds the error to the correct exclude-list or blacklist field, or to the exclude field if blacklist and exclude fields are both set)

This is basically the behavior of the code in `#validate*Blacklist()` except that the error message needs to be set to the correct field.